### PR TITLE
Add `Free Domains`

### PR DESCRIPTION
**Pull request by WilliamDavidHarrison**
## Requirements

 * [x] This is Software as a Service not self hosted
 * [x] It has a free tier not just a free trial
 * [x] The submission mentions what is free
 * [x] The submission is not already present in the list
 * [x] The service has contact details of those running it and a privacy policy

### CODE REVIEW

**Comment by WilliamDavidHarrison** [comment]
The project is literally called `Free Domains` incase that's an issue.

File: README.md [hunk 1]
```diff
@@ -1118,6 +1118,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
 
 ## Domain
   * [Freenom](https://freenom.com) — Free .tk,.ml,.ga,.cf and .gq domain.
+  * [Free Domains](https://github.com/free-domains/register) - Free subdomains for developers.
```
Comment (ripienaar) [inline README.md:1121]: Please note the exclusions when you opened the PR `* Free dns services that are generic frontends to CloudFlare or similar`

Comment (WilliamDavidHarrison) [reply README.md:1121]: @ripienaar It is a subdomain service where users can register domains such as `foo.is-a-frontend.dev`, it is not a DNS host.

Comment (WilliamDavidHarrison) [reply README.md:1121]: It is like is-a.dev and Open Domains.

Comment (ripienaar) [reply README.md:1121]: I know what it is, its a generic frontend to cloudflare dns, exactly as it says is not accepted.

Comment (WilliamDavidHarrison) [reply README.md:1121]: @ripienaar It's not a DNS host... I buy domains and let people create subdomains on them.

Comment (ripienaar) [reply README.md:1121]: Again, I know what it is, you dont need to explain.

Comment (WilliamDavidHarrison) [reply README.md:1121]: I don't understand why it's being rejected when it's the exact same as the other services.

Comment (ripienaar) [reply README.md:1121]: I also rejected several others of those already, leaving just the ones already on here.

It adds zero value, the services are not unique, you just copy each others code and put it there and come here presenting it as a SaaS when yo have like 1 user.  Generally the people who run them think its ok to come here and argue and be general nuisances, so I stopped accepting them.

Please accept the rejection wthout further arguiing

Comment (WilliamDavidHarrison) [reply README.md:1121]: Ok

**Pull request closed without merging**